packages/teleterm/src/ui/services/notifications/notificationsService.ts

```diff
@@ -1,10 +1,11 @@
 import { useStore } from 'shared/libs/stores';
 
-import { ImmutableStore } from 'teleterm/ui/services/immutableStore';
 import {
   NotificationItem,
   NotificationItemContent,
-} from 'teleterm/ui/components/Notifcations';
+} from '@gravitational/shared/components/Notification';
```

Comment:
The `@gravitational/` part isn't necessary, I think we usually just use the alias to import straight from `shared/something`. But AFAIR VSCode's refactoring tools do insert that prefix automatically.